src/JsonApi/Serializer/CollectionNormalizer.php

```diff
@@ -88,6 +88,10 @@ protected function getItemsData($object, string $format = null, array $context =
             }
 
             $data['data'][] = $item['data'];
+
+            if (isset($item['included'])) {
+                $data['included'] = array_merge($data['included'] ?? [], $item['included']);
```

Comment:
`$data['included']` is always defined because of the isset the line before, `??` can be removed.